core/module_specifier.rs

```diff
@@ -32,11 +32,19 @@ impl fmt::Display for ModuleResolutionError {
         write!(f, "invalid base URL for relative import: {}", err)
       }
       InvalidPath(ref path) => write!(f, "invalid module path: {:?}", path),
-      ImportPrefixMissing(ref specifier) => write!(
-        f,
-        "relative import path \"{}\" not prefixed with / or ./ or ../",
-        specifier
-      ),
+      ImportPrefixMissing(ref specifier, ref maybe_referrer) => {
+        let msg = format!(
+          "relative import path \"{}\" not prefixed with / or ./ or ../.",
```

Comment:
I think the trailing period here is confusing. I suggest removing it.

Reply:
How about:
```
Uncaught ImportPrefixMissing: relative import path "baz" from "file:///type_definitions/bar.d.ts" is not prefixed with / or ./ or ../ 
```
?